### ISSUE TITLE
InfoBar status text not read by JAWS

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6965]
### System details

    RStudio Edition : Server
    RStudio Version : 1.3.959
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- start JAWS screen reader and load RStudio Server in Chrome
- open a source file that references an uninstalled package
- for example, has `library(progress)` but the `progress` package isn't installed]
- the source editor shows an infobar saying "Package progress required but is not installed."

### Describe the problem in detail
- the text of that infobar is not announced by JAWS despite it being marked with status role
- it is announced by VoiceOver on Safari and NVDA on Chrome

### Describe the behavior you expected
- should be announced

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by kfeinauer** [comment]
@gtritchie Are you able to get this info bar to even pop up on the `access2.soleng.rstudioservices.com` server? I cannot get the info bar to show up indicating my package is not installed, so having a hard time verifying this ticket.

**Comment by kfeinauer** [comment]
Verified fixed in RSP 1.4.953-1
